rinex/src/navigation/health/qzss.rs

```diff
@@ -47,48 +44,77 @@ impl QZSSHealth {
 }
 
 bitflags! {
-    /// QZSS Legacy Sv Health indication.
+    /// QZSS LNAV SV Health indication.
     /// See [Bibliography::RINEX3] and [Bibliography::QzssPnt] 5.4.1
     /// for more information.
     #[derive(Default, Debug, Clone, PartialEq, PartialOrd)]
     #[cfg_attr(feature = "serde", derive(Serialize))]
-    pub struct LegacyHealth: u64 {
-        const L1CA_B= 0x01 << 22;
-        const L1C_A = 0x01 << 21;
-        const L2    = 0x01 << 20;
-        const L5    = 0x01 << 19;
-        const L1C   = 0x01 << 18;
-        const L1C_B = 0x01 << 17;
-
+    pub struct LNAVHealth: u64 {
```

Comment:
Changed the bit definitions as the RINEX spec is trying to say that this value is 6 bits long, taken from bit positions 22-17 of the navigation data Word 3 of Subframe 1